### ISSUE TITLE
Angular 8 upgrade

### ISSUE THREAD

**Issue by tanepiper** [UltimateAngular/angular-pro-app-seed#15]
Expected Behaviour:

To download the pro starter and be able to use the lastest Angular (8)

Actual Behaviour:

Downloading the starter kit is on Angular 4 and requires a significant upgrade.

**Comment by Danovich** [comment]
@toddmotto the curse is not usable with angularfire 5. Its completely outdated...

**Comment by ccarrero301** [comment]
Yes, Would be nice if the code gets updated to the latest angular version
